sunpy/visualization/animator/tests/test_mapsequence.py

```diff
@@ -0,0 +1,72 @@
+import os
+
+import pytest
+
+from astropy.visualization.wcsaxes import WCSAxes
+
+import sunpy.data.test
+import sunpy.map
+from sunpy.tests.helpers import figure_test
+from sunpy.visualization.animator import MapSequenceAnimator
+
+testpath = sunpy.data.test.rootdir
+
+
+@pytest.fixture
+def test_map_sequence():
+    return sunpy.map.Map(
+        os.path.join(
+            testpath, 'aia_171_level1.fits'), os.path.join(
+            testpath, 'mdi_fd_Ic_6h_01d.5871.0000_s.fits'), sequence=True)
+
+
+def test_construct_map_sequence_animator(test_map_sequence):
+    map_animator = MapSequenceAnimator(test_map_sequence)
+    assert isinstance(map_animator, MapSequenceAnimator)
+
+
+@figure_test
+def test_map_sequence_animator_wcs_simple_plot(test_map_sequence):
+    map_animator = MapSequenceAnimator(test_map_sequence)
+    return map_animator.fig
```

Comment:
What are this test (and all others that are "returning") testing?

Reply:
I've tested this class similar to the ```arrayanimator``` class where the basic plots are just being tested, was that wrong?